index.html

```diff
@@ -167,7 +167,7 @@ <h2>Multi-Post Stories</h2>
 
     <section class="about">
         <div class="deskAbout">
-            <h1>About<br>Myself</h1>
+            <h2>About<br>Myself</h2>
             <p>Hello I'm a software developer! I can help you build a product , feature or website Look
```

Comment:
- [ ] From line no. 177 to 188 - All the image `alt` text is similar. Please give alt with appropriate alternative text for all the image.
- [ ] From line 235 to 241 - All input fields are missing form label. Please add label to each input field to resolve this accessibility issue.
- [ ] On line 44, 175, and 226 all the text has contrast issue. Please fix all these three text to make your website more accessible for all kind of users. 

You can check all these issue in the [aditus tool](https://www.aditus.io/button-contrast-checker/) and [WAVE tool extension](https://wave.webaim.org/).